packages/legend-studio/src/components/editor/edit-panel/service-editor/ServiceTestEditor.tsx

```diff
@@ -683,13 +689,45 @@ export const ServiceTestEditor = observer(
                   </div>
                 </div>
                 <div className="panel__header__actions">
+                  <div
+                    className={clsx(
+                      'panel__content__form__section__toggler',
+                      'service-execution-editor__test-data__hash-btn',
+                      {
+                        'panel__content__form__section__toggler--disabled':
+                          isReadOnly,
+                      },
+                    )}
+                    onClick={toggleHashTestData}
+                  >
+                    Hash Data
```

Comment:
For this, we probably can be subtle and use a toggler icon instead of a full button with text, can we find some icon that means `hash/anonymize/security/password` or something?

Reply:
No, I think it makes it clearer is we just say what is does.